assets/terraform-modules/packet/flatcar-linux/kubernetes/outputs.tf

```diff
@@ -35,10 +35,18 @@ output "calico_values" {
   value = module.bootkube.calico_values
 }
 
+output "calico-host-protection_values" {
+  value = join("", local_file.calico_host_protection.*.content)
```

Comment:
Do we need join here? Host protection is not optional, right?

Reply:
Right, I'll remove.